Deploy-CAM.ps1

```diff
@@ -2648,7 +2648,7 @@ function Confirm-ModuleVersion()
     $AzureRMModule = Get-Module -ListAvailable -Name "AzureRM"
     if ( $AzureRMModule ) {
         # have an AzureRM version - check that.
-        if ( [version]$AzureRMModule[0].Version.ToString() -lt [version]$MinAzureRMVersion) {
+        if (( $AzureRMModule.Version -ge [version]$MinAzureRMVersion) -eq $False) {
```

Comment:
This checks the whole array of module versions?

Reply:
Yep, we get the version list, then filter it for everything that is greater than the version number. If there is something over the version number it will be found in there, if not then it will read as false. So we check for a match with false. 
Its very similar to the previous iteration, just instead of finding the top result, it finds all the matches.
The matching to false is because when returning a version that isn't actually 'true' so I can't -not it.

Reply:
Thanks for the demo. I didn't know that operators like '.' and '-ge' can map to 'select-object' and 'where-object' which is actually kinda' cool but makes my brain hurt.